package.json

```diff
@@ -28,6 +28,7 @@
 		"onLanguage:cpp",
 		"onLanguage:css",
 		"onLanguage:csharp",
+		"onLanguage:elisp",
```

Comment:
"elisp" isn't a known language identifier: https://code.visualstudio.com/docs/languages/identifiers#_known-language-identifiers. I checked the marketplace and there seems to be one extension that labels it as "lisp" https://github.com/mattn/vscode-lisp/blob/master/package.json#L26, so I'm not sure how to proceed here.

Reply:
Huh.  Looks like there's no formal identifier specified by [lsp](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocumentItem) either 🤔

Fwiw you will need to install a VSCode extension that registers that language id in order for this to actually work.  So I'd just pick a VSCode extension that you want to use with elisp, and then use whatever language identifier that extension uses

If there is no such extension, you could either:

- Create your own simple extension that just registers the language id and associates it with a file extension
- Add a `contributes.languages` block to this extension.  Slightly mixed feelings about that, as it feels out of scope for this extension, but maybe the simplest path forward

Reply:
just wanted to follow up